_includes/header.html

```diff
@@ -23,6 +23,7 @@
             <a class="page-link" href="/about/">About</a>
             <a class="page-link" href="/en/blog/">Blog</a>
             <a class="page-link" href="/en/newsletters/">Newsletters</a>
+            {% if jekyll.environment == 'future' %}<a class="page-link" href="/en/topics/">Topics</a>{% endif %}
```

Comment:
With the new link, this now makes the links drop onto a different line from the logo:

![image](https://user-images.githubusercontent.com/1063656/65901426-3244c000-e386-11e9-9032-cfdc094d5f3e.png).

I don't think that looks as good. Any suggestions to make it look better?

Reply:
We can create a page `/en/publications/` (or something), have that briefly describe the type of content we publish in blog posts and newsletters, have that page link to the blog post and newsletter index pages, and then replace the Blog and Newsletters links in the header with a single Publications link.

We could also do the same thing for the two indexes we have now (Compatibility and Topics).

Alternatively or in addition, we can move the About link to the footer.  Maybe the Workshops and Dashboard links too.

Reply:
Let's start with just moving blog posts and newsletters to a publications header.

What are your thoughts about having a drop-down menu for publications, life the FAQS header at https://bitcoincore.org/ ?

Reply:
I'm meh on one-level deep drop down menus (like the FAQs header you mentioned) and opposed to >1 deep menus (like BitcoinCore.org's Development→RPC Docs→`<version>` menu; I have a hard time moving my mouse steadily enough to access that menu and I don't even have any coordination disabilities).  I'd prefer not to be the person who adds 1-deep menus to this site but I won't object to someone else doing it (assuming a sane implementation and preferably one that avoids JS).

Reply:
Meh on one-level deep and opposed to >1 deep menus seems reasonable to me. Let's keep this as is and only add drop-down menus later if the number of sections changes and we can't think of anything better.